src/clients/order/create/types.ts

```diff
@@ -0,0 +1,183 @@
+// API version: 5d077b6f-61b2-4b3a-8333-7a64ee547448
+
+import { Phone } from '@src/clients/commonTypes';
+import { MercadoPagoConfig } from '@src/mercadoPagoConfig';
+import { ApiResponse, Options } from '@src/types';
+
+export declare type CreateOrderClient = {
+  body: CreateOrderRequest,
+  config: MercadoPagoConfig
+};
+
+export declare type CreateOrderData = {
+  body: CreateOrderRequest;
+  requestOptions?: Options;
+}
+
+export declare type CreateOrderRequest = {
+  type?: string;
+  external_reference?: string;
+  transactions?: TransactionsRequest;
+  payer?: PayerRequest;
+  total_amount?: string;
+  type_config?: TypeConfig;
+  processing_mode?: string;
+  description?: string;
+  marketplace?: string;
+  marketplace_fee?: string;
+  items?: Item[];
+  expiration_time?: string;
+}
+
+export declare interface OrderResponse extends ApiResponse {
+  id?: string;
+  type?: string;
+  external_reference?: string;
+  site_id?: string;
+  created_date?: string;
+  last_updated_date?: string;
+  status?: string;
+  status_detail?: string;
+  transactions?: TransactionsResponse;
+  payer?: PayerResponse;
+  type_config?: TypeConfig;
+  total_amount?: string;
+  processing_mode?: string;
+  description?: string;
+  marketplace?: string;
+  marketplace_fee?: string;
+  items?: Item[];
+  expiration_time?: string;
+}
+
+export declare type TypeConfig = {
+  capture_mode?: string;
+}
+
+export declare type TransactionsRequest = {
+  payments?: PaymentRequest[];
+}
+
+export declare type TransactionsResponse = {
+  payments?: PaymentResponse[];
+  refunds?: Refund[];
```

Comment:
Manter a nomenclatura
```suggestion
  refunds?: RefundResponse[];
```